docs/errors.md

```diff
@@ -0,0 +1,194 @@
+# SystemJS Errors
+
+This page lists the SystemJS errors that you may encounter.
+
+## 1
+
+### Import Map contains invalid JSON
+
+SystemJS Error #1 occurs when you have a `<script type="systemjs-importmap"></script>` element in your HTML page that has invalid JSON content (See [import maps documentation](https://github.com/systemjs/systemjs/blob/master/docs/import-maps.md)).
+
+A common mistake that causes this a trailing comma on the last module in the import map, which is invalid JSON.
+
+```html
+<!-- Make sure the JSON content is valid! -->
+<script type="systemjs-importmap">
+  {
+    "imports": {
+      "foo": "/foo.js",
+      "bar": "/bar.js"
+    }
+  }
+</script>
+```
+
+Note that this error also can occur for external import maps (those with `src=""` attribute). Check the network tab of your browser devtools to verify that the response body for the external import map is valid json.
+
+## 2
+
+### Unable to resolve specifier
```

Comment:
We can make this `Unable to resolve bare specifier` as it will always relate to bare specifier resolution.